signatures/directory_listing

```diff
@@ -0,0 +1,13 @@
+name: Directory Listing
+method: GET
+status: 200
+body: 
+  regex: >
+    ^(.*)<html>
+     <head>
+      <title>Index of /(.*)</title>
+     </head>
+     <body>
+    <h1>Index of /(.*)</h1>
+      <table>(.*)$
+
```

Comment:
What about something like this?

```yaml
name: Directory Listing

request:
  method: GET

response:
  status: 200
  body: 
    regex: >
      ^(.*)<html>
       <head>
        <title>Index of /(.*)</title>
       </head>
       <body>
      <h1>Index of /(.*)</h1>
        <table>(.*)$
```

It would be good to break out the request and response portions. This would allow the author of a signature to specify when it should be applied. There are a handful of characteristics that would be important.

* http-verb
* url contains or equals
* header exists
* request body contains
* response status code
* response header exists
* body contains
* body length is greater or less than XYZ

There will be less responses to check as more criteria is defined in the request section.

Reply:
Thanks, I will remember that. This code is old, I haven't worked on this for a while and it looks bad. I will refactor the whole thing when I'm back to work